privacy-snapshot.json

```diff
@@ -72,5 +72,6 @@
   "unresponsive-rpc.test",
   "unresponsive-rpc.url",
   "user-storage.api.cx.metamask.io",
-  "www.4byte.directory"
+  "www.4byte.directory",
+  "verify.walletconnect.com"
```

Comment:
This was added as MetaMasks Test dApp makes a websocket connnection as shown in the screenshot

<img width="1840" alt="Screenshot 2024-12-10 at 3 33 47 PM" src="https://github.com/user-attachments/assets/4751a812-3d59-4b78-b6ef-3e5ddf793db5">